components/IndoorDirections/TypesOfDirections/SameFloorDirections.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Line, G } from 'react-native-svg';
+import { ClassGraph } from '../../../constants/ClassGraph';
+import { dijkstra, getFloorNumber, changeClassName } from '../Dijkstra/DijkstraAlgorithm';
+
+export function SameFloorDirections(props) {
+    const rooms = props.rooms;
+    const graph = ClassGraph();
+    const from = changeClassName(props.from);
+    const to = changeClassName(props.to);
+    const route = dijkstra(graph, from, to).path;
+
+    var getNextRoom = (index) => {
```

Comment:
getNextRoom gets the "destination" room, correct?

Reply:
dijkstra returns an array of the classes to go through. `getNextRoom` returns the next room to go to so i can draw a line from point 1 to next room to go to